src/explorer/appServiceNodes.ts

```diff
@@ -83,8 +83,8 @@ export class AppServiceNode extends NodeBase {
         
         return [
             new DeploymentSlotsNode(this.site, this.subscription),
-            new FilesNode('Files', '/site/wwwroot', this.site, this.subscription),
-            new FilesNode('Log Files', '/LogFiles', this.site, this.subscription),
+            // new FilesNode('Files', '/site/wwwroot', this.site, this.subscription), 
```

Comment:
Sorry - I meant add a comment to the file itself. Like literally add this in the file:
`// https://github.com/Microsoft/vscode-azureappservice/issues/45`